tests/conftest.py

```diff
@@ -339,6 +340,46 @@ def timezone(tzinfo):
     return tzinfo
 
 
+timed_out = False
+
```

Comment:
Can you leave a comment somewhere around here explaining why we do this "manually" instead of using `pytest-timeout`?